.github/workflows/mdbook.yaml

```diff
@@ -24,7 +24,8 @@ jobs:
     - uses: actions/checkout@v2
       with:
         ref: gh-pages
-    - run: rm -rf *
+    # do not remove helm chart related files if present
+    - run: ls | grep -v -E "index.yaml|$(basename $(git rev-parse --show-toplevel))-.*\.tgz" | xargs rm -rf
```

Comment:
Is there a possibility to put tgz files in the gh-pages other than tenet-*.tgz?

Reply:
We could technically hardcode this to tenet-*.tgz but I figured this expression would be more flexible, for instance in the (unlikely) event the repository gets renamed.

Reply:
I thought you considered not deleting tgz files other than tenet-*.tgz by `$(basename $(git rev-parse --show-toplevel))-.*\.tgz`.  To me `.*\.tgz` is fine, if we never put those files.

Reply:
I was trying to avoid excluding all .tgz files from deletion, but for the forseeable future it is unlikely that .tgz files other than helm charts will get committed to gh-pages so I'll commit your suggestion.